compiler-core/src/ast.rs

```diff
@@ -1172,6 +1173,10 @@ impl SrcSpan {
     pub fn contains(&self, byte_index: u32) -> bool {
         byte_index >= self.start && byte_index < self.end
     }
+
+    pub fn contains_span(&self, other: Self) -> bool {
```

Comment:
Is this used? I couldn't see it

Reply:
Oops, forgot to remove this after testing